config/registry/aws/modules/cloudfront-distribution.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "cloudfront-distribution"
+linkTitle: "cloudfront-distribution"
+date: 2021-11-16
+draft: false
+weight: 1
+description: Set up a cloudfront distribution to serve static files from your s3 bucket
+---
+
+This module sets up a cloudfront distribution for you, tailored towards serving static websites/files from an Opta s3 
```

Comment:
Reword
```
This module sets up a cloudfront distribution. Currently, it's not very useful unless you link it with a s3 module.
```

Reply:
er... that's incorrect. It FAILS unless it's linked to an S3 bucket